pallets/collective/src/migrations/v4.rs

```diff
@@ -0,0 +1,153 @@
+// This file is part of CORD – https://cord.network
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// Copyright (C) Dhiway Networks Pvt. Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Adapted to meet the requirements of the CORD project.
+
+// CORD is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// CORD is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with CORD. If not, see <https://www.gnu.org/licenses/>.
+
+use sp_io::hashing::twox_128;
+
+use super::super::LOG_TARGET;
+use frame_support::{
+	traits::{
+		Get, GetStorageVersion, PalletInfoAccess, StorageVersion,
+		STORAGE_VERSION_STORAGE_KEY_POSTFIX,
+	},
+	weights::Weight,
+};
+
+/// Migrate the entire storage of this pallet to a new prefix.
+///
```

Comment:
@smohan-dw Considering we would be starting fresh with this network, do you foresee a 'migration' requirement? Not clear on why this would be used.

Reply:
agree. safe to remove the migration code.

Reply:
Ok, lets keep it for now, and remove as a separate PR later.

Reply:
Would love to raise a follow up PR removing migration code.